eisl.h

```diff
@@ -30,7 +30,12 @@
 #define CTRLSTK 200
 #define BACKSIZE 30
 #define EISL_PATH_MAX 256
+#if defined(__TINYC__) && defined(__riscv)
+// TCC in RISC-V cannot handle a larger number
+#define COL_SIZE 51
```

Comment:
This seems to be the number of columns for command-line editing. If the curses & REPL editor feature was stripped out cleaner I doubt it's needed at all for WITHOUT_CURSES=1 builds.

Reply:
You are most likely right, but I did not have time to make sure that was the case. Nonetheless, I know that if `ncurses` support is indeed there `make CC=tcc` does fail without that line. So this allows TCC in RISC-V with curses support to work :)